modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -786,7 +924,7 @@ FeatureFloodCount::visitElementalNeighbors(const Elem * elem, unsigned long curr
   std::vector<const Elem *> all_active_neighbors;
 
   // Loop over all neighbors (at the the same level as the current element)
-  for (unsigned int i = 0; i < elem->n_neighbors(); ++i)
+  for (auto end = elem->n_neighbors(), i = 0u; i < end; ++i)
```

Comment:
This is at least the third distinct style of counter based for loop. I understand why one has auto and the other one has decltype only, but maybe we should just stick to one style(?)